### PR TITLE
Feature/ocr info detection

### PR DESCRIPTION
**Pull request by moshesab**

### CODE REVIEW

File: src/app/ui/upload/upload.component.ts [hunk 1]
```diff
@@ -37,6 +40,7 @@ enum UploadState {
   templateUrl: './upload.component.html',
   styleUrls: ['./upload.component.scss']
 })
+@Injectable()
```
Comment (ron4548) [inline src/app/ui/upload/upload.component.ts]: Why did you add it? a component shouldn't be injectable..

File: package.json [hunk 2]
```diff
@@ -25,6 +25,7 @@
     "@fortawesome/angular-fontawesome": "^0.3.0",
     "@fortawesome/fontawesome-svg-core": "^1.2.17",
     "@fortawesome/free-solid-svg-icons": "^5.8.1",
+    "@google-cloud/vision": "^0.25.0",
```
Comment (ron4548) [inline package.json]: Looks like you didn't used this import at all, so delete it

File: src/app/ui/upload/upload.component.ts (hunk 1 above)
Comment (ameeramer) [reply src/app/ui/upload/upload.component.ts]: I looked online for using httpClient and that was the example, with an injectable, now I understand it is not related to the httpClient, but only there for the example. I will remove it.
